CHANGELOG.md

```diff
@@ -19,8 +19,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 ### Enhancements
 - Substitute REST path or body parameters in Workflow Steps ([#525](https://github.com/opensearch-project/flow-framework/pull/525))
 - Added an optional workflow_step param to the get workflow steps API ([#538](https://github.com/opensearch-project/flow-framework/pull/538))
+- Enable Flow Framework by default ([#553](https://github.com/opensearch-project/flow-framework/pull/553))
```

Comment:
May be under `Maintenance` or `Refactoring`?

Reply:
I had that same thought process about where it should go.

It's an enhancement for users who don't have to enable it manually, or so I concluded.